Monika After Story/game/script-topics.rpy

```diff
@@ -15749,3 +15749,52 @@ label monika_literature_value:
     m 1eku "Honestly, I think if more people valued books and poems a little more, the world would be a much better place."
     m 1hksdlb "That's just my opinion as president of a literature club, though. {w=0.2}I guess most people wouldn't think that deeply about it."
     return
+    
+# Do you care about the environment?
+default persistent._mas_pm_cares_environment = None
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_environment",category=['nature', 'life'],prompt="The environment",random=True))
+ 
+label monika_environment:
+    m 2esd "Hey [player], I've been wondering..."
+    $ _history_list.pop()
+    menu:
+        m "Do you care about the environment{fast}?"
+        "Yes.":
+            $ persistent._mas_pm_cares_environment = True
+            m 2suo "Really? That's wonderful!"
+            m 2wsa "I'm so glad to hear that, [player]."
+            m 3esd "Personally, I think that the environment is something we should treasure."
+            m 3esa "It's a beautiful world filled with all kinds of life, and there's a lot nature does to lend humanity a hand."
+            m 4esb "Insects pollinate crops, decomposers recycle waste, trees provide food and shade, oceans absorb heat, and so on."
+            m 1esu "Since nature does so much to help us, I think it's only fair that we give something back in return."
+            m 7hub "So, here's Monika's Green Tip of the Day!"
+            m 7esc "Sometimes, it might seem difficult to make a difference when products such as electric vehicles and solar roofs can be expensive."
+            m 3eub "However, you can make a difference and save money just by performing a few simple tasks each day!"
+            m 3esb "Turning off appliances, taking shorter showers, buying a reusable water bottle, commuting by public transport..."
+            m 4esa "If you feel up to it, you can even become a vegetarian and start growing your own garden!"
```

Comment:
I think it's fine to suggest growing your own garden for food, but I don't see her actively trying to encourage you to become a vegetarian.

Reply:
She talks about wanting you to adopt a "healthier lifestyle" in monika_soda, monika_fastfood, and one of the time concern prompts (ctrl + F lifestyle), so I think she'd at least suggest it (she is genuinely concerned about the player's health, after all). Maybe something like this would sound less like "encouragement" and more like a suggestion:

> ...you could even try growing your own garden or becoming a vegetarian

She certainly wouldn't force you to become a vegetarian, but I think she'd probably bring it up in some manner. Either way, I'll sleep on it.

Reply:
You can be healthy without being a vegetarian. Maybe this is just a reference to her being a vegetarian herself, but she would certainly respect your decision to be or not to be a vegetarian, and I don't know if she would push it like this.

Reply:
What I'm suggesting is removing that part about her suggesting you become a vegetarian, or making it sound a little lighter.